qm-plugins/qm-object-cache/css/style.css

```diff
@@ -1,64 +1,69 @@
+#query-monitor-main .qm-hide-operation, #query-monitor-main .qm-hide-group {
+    display: none !important;
+  }
+
+/* TODO: Remove the below after deployment of get_stats() */
 #query-monitor-main #object-cache-stats h2 {
-	float: left  !important;
-	clear: none !important;
-	min-width: 150px !important;
-	border: 1px solid #eee !important;
-	padding: 5px 10px 15px !important;
-	text-align: center !important;
-	color: #000 !important;
-	font-family: georgia,times,serif !important;
-	font-size: 28px !important;
-	margin: 15px 10px 15px 0 !important;
+  float: left  !important;
```

Comment:
The WordPress Coding Style is to use tab indentation for CSS files. See https://core.trac.wordpress.org/browser/trunk/.editorconfig#L14

(I'm surprised we don't have a `.editorconfig` file in the root of this repo to help catch this.)